src/App.tsx

```diff
@@ -1,61 +1,124 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { UserMas } from './interfaces/UserMas';
+import { TodosMas } from './interfaces/TodosMas';
```

Comment:
```suggestion
import { User } from './interfaces/User';
import { Todo } from './interfaces/Todo';
```
What is `Mas`? Just User, Todo, etc